advocacy_docs/partner_docs/ImpervaDataSecurityFabric/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Imperva Data Security Fabric'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Imperva       |
+| **Partner Product**   |  Imperva Data Security Fabric       |
+| **Web Site**      |  https://www.imperva.com/    |
+| **Version**   |  14.9      |
+| **Product Description**      |  Imperva Data Security Fabric assesses and helps mitigate database risks and detects compliance and security policy violations. EDB Postgres Advanced Server and PostgreSQL databases can use the Imperva Agent with Imperva Data Security Fabric to see who is accessing data, where sensitive data lives and if the data access activity is good or bad.     |
```

Comment:
Wording of the second sentence seems a bit off to me, since the database can't use Imperva, but the Customer can use Imperva with their EDB database. Try something like this instead: 

"Customers can use Imperva Data Security Fabric with their EDB Postgres Advanced Server and/or PostgreSQL databases and see where their sensitive data resides, who is accessing the data, and determine whether the user data access activity is good or bad."